streamwebs_frontend/streamwebs/templates/streamwebs/sites.html

```diff
@@ -25,34 +25,55 @@
 {% block body_title %}Sites{% endblock %}
 
 {% block content %}
-    <div id="site_search">
-        <label for="search_bar">Search sites: </label>
-        <input type="search" id="search_bar" name="search" oninput="search()" />
-        <label for="steward_box">Student Stewardship Project</label>
-        <input type="checkbox" id="steward_box" name="stewardship" value="steward" onchange="search()" />
-        <label for="salmon_box">Salmon Watch</label>
-        <input type="checkbox" id="salmon_box" name="salmon_watch" value="salmon" onchange="search()" />
-        <label for="available_box">Available Project</label>
-        <input type="checkbox" id="available_box" name="available" value="available" onchange="search()" />
-        <div id="search_dropdown">
-        {% for site in sites %}
-            <div name="{{site.slug}}" class="search_item {{site.site_type}}" style="display: none">
-                <p name="{{site.slug}}" onclick="submit_search(this)">{{site.name}}</p>
+    <div class="container">
+        <div id="site_search">
+            <div class="row">
+                <nav class="search-nav col s8 cyan">
+                    <div class="nav-wrapper">
+                        <div class="input-field">
+                            <input id="search" type="search" autocomplete="off" oninput="search()">
```

Comment:
Should probably have some placeholder text so the student/user knows what the search is for